src/weapons.h

```diff
@@ -62,7 +62,7 @@ class Weapons final : public BaseEvents
 class Weapon : public Event
 {
 	public:
-		explicit Weapon(LuaScriptInterface* interface) : Event(interface) {}
```

Comment:
Why did you remove the `explicit` specifier?

Reply:
Not really sure, I'll re add them